meta-mender-core/classes/mender-helpers.bbclass

```diff
@@ -19,6 +22,9 @@ get_uboot_device_from_device() {
         /dev/mmcblk[0-9])
             echo ${1#/dev/mmcblk}
             ;;
+        ubi[0-9])
+            echo ubi
```

Comment:
> All of this was building up to a question. Is it correct of me to re-use the mender_uboot_dev for this use-case, or should I create a new configuration option?

I think you should probably create a new option for this, since the existing variables are reserved for partitions inside UBI. There isn't an existing mechanism for telling where the "outer" MTD partitions are.

Maybe bitbake variable `MENDER_MTD_UBI_DEVICE` and U-Boot variable `mender_mtd_ubi_dev`? I guess this has to be configured separately regardless.